src/filters/array.rs

```diff
@@ -0,0 +1,171 @@
+use liquid_value::Value;
+
+use super::{check_args_len, invalid_input};
+use interpreter::FilterResult;
+
+/// Receives a `Value::Array` as an input.
+/// Returns a copy of the input with the given value appended at the end.
+pub fn push(input: &Value, args: &[Value]) -> FilterResult {
```

Comment:
Since this from jekyll and not liquid, we should put these under `extra-filters`

Granted, we need to re-look into how to handle that at some point

Reply:
Sorry, didn't catch that you did this in the mod.rs